pandas/src/generated.pyx

```diff
@@ -9110,6 +9110,8 @@ def group_count_bin_float64(ndarray[float64_t, ndim=2] out,
         ndarray[int64_t, ndim=2] nobs = np.zeros((out.shape[0], out.shape[1]),
                                                  dtype=np.int64)
 
+    if len(bins) == 0:
```

Comment:
Oh crap, my bad!  I did see the name, but then I looked at git history and saw what appeared to be people editing it (but I get it now).

Do you think it's worth adding a warning message at the top for new devs [like NumPy does](https://github.com/numpy/numpy/blob/master/numpy/distutils/conv_template.py#L57)?  Or should it be obvious from the name?